### PR TITLE
Ft/wisdom/dashboard/application

### PR DESCRIPTION
**Pull request by Ajinormotor**
## Description

Please include a summary of the changes and the related issue.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Ajinormotor** [comment]
am done with dashboard application, if any correction?

File: src/chunks/admin-template-chunk/application-template-chunk/ApplicationTemplateChunk.tsx [hunk 1]
```diff
@@ -0,0 +1,616 @@
...
+
+        <div className="border-[1px] border-[#1B1B1B66] gap-[10px] p-[10px] w-[112px] h-[40px] flex items-center justify-center">
+          <h1 className="font-[600] text-[16px] leading-[16px] font-epilogue text-[#1B1B1B]">
+            July
+          </h1>
+          <Image src={date} alt="date_icon" className="w-[18px] h-[20px]" />
+        </div>
+      </div>
+
+      <div className="w-full h-[46px] flex border-b-[1px] border-[#D6DDEB] gap-[60px]">
```
Comment (Blossom1-chike) [inline src/chunks/admin-template-chunk/application-template-chunk/ApplicationTemplateChunk.tsx:46]: image

compare this image with what you have, the font sizr, weight and style, please update.
also, move this filter tab into a component of its own and then use it here so its reusable

File: src/chunks/admin-template-chunk/application-template-chunk/ApplicationTemplateChunk.tsx [hunk 2]
```diff
@@ -0,0 +1,616 @@
...
+            Don’t give up, Tosin
+          </h2>
+          <p className="text-[#515B6F] opacity-60 font-[400] font-epilogue leading-[30px] text-[16px]">
+            Here are your applications so far
+          </p>
+        </div>
+
+        <div className="border-[1px] border-[#1B1B1B66] gap-[10px] p-[10px] w-[112px] h-[40px] flex items-center justify-center">
+          <h1 className="font-[600] text-[16px] leading-[16px] font-epilogue text-[#1B1B1B]">
+            July
```
Comment (Blossom1-chike) [inline src/chunks/admin-template-chunk/application-template-chunk/ApplicationTemplateChunk.tsx:40]: work on this implementation such that when clicked, it shows a calendar where one can select date. make it functional

File: src/chunks/admin-template-chunk/application-template-chunk/ApplicationTemplateChunk.tsx
Comment (Blossom1-chike) [inline src/chunks/admin-template-chunk/application-template-chunk/ApplicationTemplateChunk.tsx:1]: for the tables, i dont think each filter should have different data, the whole idea is that the filters same data for the different tables, so please look into the implementation, we can look through it together when you're available

**Review by Blossom1-chike — CHANGES_REQUESTED** [review]

**Comment by Ajinormotor** [comment]
i have worked on all the changes you requested;
Made the calender fucntional
Each table with the necessary filter
worked on the font size  etc

**Review by Blossom1-chike — APPROVED** [review]

**Pull request merged**